### ISSUE TITLE
telega-filter check phase failing

### ISSUE THREAD

**Issue by brettgilio** [zevlg/telega.el#185]
While attempting to upgrade telega on Guix (finally, sorry for the delay) there is an error around the check phase for telega-filters. Will look into this when I can, but opening for others to see:

```
starting phase `check'
emacs -Q -batch -L . -f package-initialize -l ert -l test.el \
         -f ert-run-tests-batch-and-exit
Running 9 tests (2020-07-09 02:22:01+0000)
   passed  1/9  telega-desurrogate
   passed  2/9  telega-emacs-setup
Test telega-filters backtrace:
  signal(ert-test-failed (((should (= (length (telega-filter-chats tel
  ert-fail(((should (= (length (telega-filter-chats telega--ordered-ch
  (if (unwind-protect (setq value-231 (apply fn-229 args-230)) (setq f
  (let (form-description-233) (if (unwind-protect (setq value-231 (app
  (let ((value-231 (quote ert-form-evaluation-aborted-232))) (let (for
  (let* ((fn-229 (function =)) (args-230 (condition-case err (let ((si
  (closure (t) nil (let* ((fn-219 (function not)) (args-220 (condition
  ert--run-test-internal(#s(ert--test-execution-info :test #s(ert-test
  ert-run-test(#s(ert-test :name telega-filters :documentation "Test `
  ert-run-or-rerun-test(#s(ert--stats :selector t :tests [#s(ert-test 
  ert-run-tests(t #f(compiled-function (event-type &rest event-args) #
  ert-run-tests-batch(nil)
  ert-run-tests-batch-and-exit()
  command-line-1(("-L" "." "-f" "package-initialize" "-l" "ert" "-l" "
  command-line()
  normal-top-level()
Test telega-filters condition:
    (ert-test-failed
     ((should
       (=
	(length ...)
	(1- ...)))
      :form
      (= 3 2)
      :value nil))
   FAILED  3/9  telega-filters
   passed  4/9  telega-formatting
   passed  5/9  telega-info
   passed  6/9  telega-internationalization
   passed  7/9  telega-server-json-parsing
   passed  8/9  telega-tme-open
   passed  9/9  telega-waveform

Ran 9 tests, 8 results as expected, 1 unexpected (2020-07-09 02:22:01+0000)

1 unexpected results:
   FAILED  telega-filters

make: *** [Makefile:13: test] Error 1

Test suite failed, dumping logs.
command "make" "test" "-j" "4" "WITH_VOIP=t" failed with status 2
builder for `/gnu/store/7hg7ap3scq0qp5f0vli4wcvlndl31m44-emacs-telega-0.6.26-1.505f5cb.drv' failed with exit code 1
build of /gnu/store/7hg7ap3scq0qp5f0vli4wcvlndl31m44-emacs-telega-0.6.26-1.505f5cb.drv failed
View build log at '/var/log/guix/drvs/7h/g7ap3scq0qp5f0vli4wcvlndl31m44-emacs-telega-0.6.26-1.505f5cb.drv.bz2'.
guix build: error: build of `/gnu/store/7hg7ap3scq0qp5f0vli4wcvlndl31m44-emacs-telega-0.6.26-1.505f5cb.drv' failed
```

**Comment by brettgilio** [comment]
Can confirm this is a regression since 0.6.23.

**Comment by brettgilio** [comment]
Fixed with #189, closing.
